modules/images-editing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-editing-image-registry-settings-hcp_{context}"]
+= Editing image registry settings for {hcp-title}
+
+You can change the image registry config with the `rosa edit` command.
+
+[WARNING]
+====
+When the `allowedRegistries` parameter is defined, all registries, are blocked unless explicitly listed. To prevent pod failure, a list of Red Hat registries is automatically whitelisted, as they are required by payload images within your environment. The current list consists of `image-registry.openshift-image-registry.svc:5000,quay.io,registry.redhat.io` and it is also visible when running `rosa describe cluster`.
+====
+
+[NOTE]
+====
+You can change any registry related parameter which will trigger a rollout across all machinepools (all machinepool nodes will be recreated, following pod draining from each node).
+====
+
+.Procedure
+
+* Update or edit the image registry for the cluster by running the following command:
+
++
+[source,terminal]
+----
+$ rosa edit cluster --registry-config-insecure-registries <insecure_registries> \
+   --registry-config-allowed-registries <allowed_registries> \
+   --registry-config-allowed-registries-for-import <registry_name:insecure> \
+   --registry-config-additional-trusted-ca <additional_trusted_ca_file>
+----
++
+.Example output
+[source,terminal]
+----
+rosa edit cluster -c <cluster_name> --registry-config-allowed-registries <allowed_registries> \
+? Changing any registry related parameter will trigger a rollout across all machinepools \
+(all machinepool nodes will be recreated, following pod draining from each node). \
+Do you want to proceed? Yes \ <1>
+I: Updated cluster '<cluster_name>'
+----
+
+.Verification
+* Run the `rosa describe` command again, to see if the changes you made to your image registry updated by running the following command:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+Name:                       rosa-hcp-test
+Domain Prefix:              rosa-hcp-test
+Display Name:               rosa-hcp-test
+ID:                         <cluster_hcp_id>
+External ID:                <cluster_hcp_id>
+Control Plane:              ROSA Service Hosted
+OpenShift Version:          4.Y.Z
+Channel Group:              stable
+DNS:                        <dns>
+AWS Account:                <AWS_id>
+AWS Billing Account:        <AWS_id>
```

Comment:
I'll stop commenting on these because there are a lot more, but throughout the PR, change any `<AWS_id>` to lowercase `<aws_id>`, unless this is verbatim text from the actual output. I'm assuming though that this is just scrubbing the real AWS ID out for the sake of the example output.
```suggestion
AWS Account:                <aws_id>
AWS Billing Account:        <aws_id>
```

Reply:
replaced in 24 places in the 2 files, thanks